src/app/components/play-round/play-round.component.scss

```diff
@@ -37,6 +36,13 @@
   justify-content: space-around;
   align-content: space-around;
   align-items: center;
+
+  // Scale the font's point size up to 6% of the average of the viewport width
+  // and the viewport height. (Never scale it down, though.)
+  // This makes the pollen indicators look pretty good in both landscape and
+  // portrait mode across a range of devices.
+  font-size: max(calc(6 * (1vw + 1vh) / 2), 100%);
+  padding: 0.6em 0.4em;
```

Comment:
Do we still want to be scaling this proportionally to width?

Reply:
I think so—the FAB only has to be as large as someone's thumb, so we can get away with a one-size-fits-all approach, but the pollen counter needs to be prominent, so we'll want it to take up more area on larger screens.

I'm definitely open to other ways doing that scaling, though!